Digital_guj_scraper.py

```diff
@@ -13,12 +13,11 @@
 
 
 # smtp connection
-def Sendemail(date,Notification, link,receivers_email):
+def Sendemail(date,Notification, link):
```

Comment:
Can you please improve mail text by removing this unwanted texts.

![image](https://github.com/Rjchauhan18/gtu_notification/assets/113838908/7c52297c-f0f9-4162-b244-581e1147e163)

Reply:
hello, can you please explain what you mean by unnecessary text in this image? All of them are present in the notification part of the webpage